draft/spec/index.html

```diff
@@ -340,6 +340,47 @@ <h2>Object Structure</h2>
         ├── inventory.json
         └── inventory.json.sha512
         </pre>
+
+        <section id="object-interoperability">
+            <h2>Interoperability</h2>
+            <p>
+                Within an OCFL Object and its associated <a>Inventory</a>, the following restrictions are applied
+                to filesystem features. These restrictions are provided in an attempt to normalize file handling
+                behaviors and addressability of files within the Inventory.
+            </p>
+            <p>
+                If use of these features are required to preserve an exact copy of the content (for example,
+                preserving a hard-drive image), implementers SHOULD consider using a packaging format,
+                such as ZIP or TAR files or ISO disk images, to preserve the content. See the
+                [[OCFL-Implementation-Notes]] for further details.
+            </p>
+            <ul>
+                <li>
+                    <strong>Symbolic links</strong> (aka 'shortcuts' or 'aliases') MUST NOT be used within an
+                    OCFL Object. These are not portable and may lead to undefined behaviours in client applications
+                </li>
+                <li>References to <strong>file paths</strong> within the <a>inventory</a> MUST use the "forward-slash"
+                    ('/') as a path separator. Some filesystem use other characters such as the "back-slash" ('\')
+                    or "colon" (':') as a path delimiter.
+                </li>
+                <li>
+                    <strong>File permissions</strong> MAY be applied to files in an OCFL Object; however, implementers
+                    should be aware that these are not portable across filesystems, and so should not be relied on
+                    to protect sensitive content. This is also the case for extended filesystem flags, such as
+                    Access Control Lists or Hidden files.
+                </li>
+                <li>
+                    The <strong>character encoding</strong> of the filesystem and Inventory SHOULD be
+                    Unicode-compatible, either UTF-8, UTF-16, or UCS-2. Implementers may experience problems
+                    integrating files from filesystems that use other encodings.
+                </li>
+                <li>
+                    Some filesystems are not <strong>case sensitive</strong>, meaning two file names that differ only
```

Comment:
Can we add a normative `MUST`, `SHOULD`, or `MAY` in this last item?

Reply:
This seems like a significant question. We either allow a degree of incompatibility or we are rather restrictive. Neither is very appealing. Bagit takes the first approach but gives a description of the issue: https://tools.ietf.org/html/draft-kunze-bagit-17#section-6.1.1.1
so I think I would tend toward that approach. A link to BagIt might be nice here but essentially this is the tack @ahankinson has taken in the PR